### PR TITLE
mongodb_shard: fix doc formatting

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
mongodb_shard: fix doc formatting

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
```lib/ansible/modules/database/mongodb/mongodb_shard.py```

### CODE REVIEW

**Comment by Akasurde** [comment]
cc @felixfontein

File: test/sanity/ignore.txt [hunk 1]
```diff
@@ -2822,9 +2822,6 @@ lib/ansible/modules/database/mongodb/mongodb_parameter.py validate-modules:param
 lib/ansible/modules/database/mongodb/mongodb_replicaset.py use-argspec-type-path
 lib/ansible/modules/database/mongodb/mongodb_replicaset.py validate-modules:parameter-list-no-elements
 lib/ansible/modules/database/mongodb/mongodb_shard.py use-argspec-type-path
```
Comment (Akasurde) [inline test/sanity/ignore.txt:2824]: I think we can remove this as well.

Comment (Akasurde) [reply test/sanity/ignore.txt:2824]: nvm

**Review by felixfontein — APPROVED** [review]
shipit

**Pull request merged**